include/RED4ext/Scripting/Natives/animIAnimationBuffer.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <RED4ext/Common.hpp>
+#include <RED4ext/ISerializable.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/AnimDataAddress.hpp>
+
+namespace RED4ext
+{
+namespace anim
+{
+struct IAnimationBuffer : ISerializable
+{
+    static constexpr const char* NAME = "animIAnimationBuffer";
+    static constexpr const char* ALIAS = NAME;
+
+    virtual bool Initialize() = 0;
+    virtual bool Load();
```

Comment:
I see the the rest of the class is virtual, do you need to define these methods then?

Reply:
Are you talking about defining the methods in general for IAnimationBuffer? Or the specific implementations?

If it's the latter case those are methods I found to have some level of base implementation, which I assumed could be relevant. But in hindsight it seems unnecessary

Reply:
The specific implementation for these methods https://github.com/WopsS/RED4ext.SDK/pull/157/files#diff-02e0f54aa59dc27be0144120e671cfde17e58c320b3d958b416466509d0605b9R7-R82.

Reply:
@Mozz3d can you remove these implementations?